cmd/metacache-set.go

```diff
@@ -553,7 +553,7 @@ func (er *erasureObjects) listPath(ctx context.Context, o listPathOptions, resul
 
 	// Special case: ask all disks if the drive count is 4
 	if askDisks <= 0 || er.setDriveCount == 4 {
-		askDisks = len(disks)                // with 'strict' quorum list on all online disks.
+		askDisks = (len(disks) + 1) / 2      // with 'strict' quorum list on read quorum disks.
```

Comment:
But it is no longer strict, and a version would have to be on *all* randomly selected drives to be listed.

AFAICT this will make listing much more un-reliable, and definitely not "strict" (which is the askDisk <= 0).

Reply:
@klauspost correct but we are trying to fix the connection load issue with the customer with 1152 disks you should see how many lists calls they are doing - this is simply overloading the system.

Strict is enough to be 8 disks we do not need it to be all 16 since there is enough room. We also have `fallback() code to ensure that we fallback in-case things fail but that's all that is needed.